test/langtools/jdk/javadoc/lib/javadoc/tester/JavadocTester.java

```diff
@@ -543,6 +557,26 @@ private void checkOutput(String path, String fileString, boolean expectedFound,
         }
     }
 
+    private static boolean checkIfPrefixes(String... strings) {
+        String[] copy = Arrays.copyOf(strings, strings.length);
+        Arrays.sort(copy);
+        for (int i = 0; i < copy.length - 1; i++) {
+            if (isFirstPrefixOfSecond(copy[i], copy[i + 1]))
+                return true;
+        }
+        return false;
+    }
```

Comment:
Interesting trade off in performance:
* either create sorted array and do `o(N)` scan 
* or just do `o(N^2)` scan

Reply:
For manually enumerated strings the difference between O(N*log(N)) and O(N^2) is all but absent due to a rather small N. That said, if you think that O(N^2) scan would read better, I can use it instead.

Reply:
I don't have any data to support a strong feeling either way.  It did take a moment to realize what the code was doing.